docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+| kind                     | :ref:`livelinessqospolicykind`| AUTOMATIC         |
++--------------------------+-------------------------------+-------------------+
+| lease_duration           | fastrtps::Duration_t          | c_TimeInfinite    |
++--------------------------+-------------------------------+-------------------+
+| announcement_period      | fastrtps::Duration_t          | c_TimeInfinite    |
++--------------------------+-------------------------------+-------------------+
+
+* **Kind**: This data member establishes if the service needs to assert the liveliness automatically or if it needs
+  to wait until the liveliness is asserted by the publishing side.
+  See :ref:`livelinessqospolicykind` for further details.
+* **Lease Duration**: Amount of time to wait since the last time the DataWriter asserts its liveliness to consider
+  that it is no longer alive.
+* **Announcement Period**: Amount of time between consecutive liveliness messages sent by the DataWriter.
+  This data member only takes effect if the kind is AUTOMATIC or MANUAL_BY_PARTICIPANT and needs to be lower than the
+  `lease_duration`.
+
+.. note::
+     This QoS Policy concerns to Topic, DataReader and DataWriter entities.
+     :raw-html:`<br />`
+     It cannot be changed after the creation of the entities.
+
+.. warning::
+    For DataWriters and DataReaders to match, they must follow the compatibility rule. See :ref:`liveliness_compatibilityrule`
+    for further details.
+
+.. _livelinessqospolicykind:
+
+LivelinessQosPolicyKind
+"""""""""""""""""""""""
+
+There are three possible values:
+
+* **Automatic**: The service takes the responsibility for renewing the leases at the required rates, as long as the
+  local process where the participant is running and the link connecting it to remote participants exists, the
+  entities within the remote participant will be considered alive.
+  This kind is suitable for applications that only need to detect whether a remote application is still running.
+* The two **Manual** modes require that the application on the publishing side asserts the liveliness periodically
+  before the lease_duration timer expires. This action can be done by calling `assert_liveliness` member function, or
+  by writing some data.
+
+  * **Manual by Participant**: If one of the entities in the publishing side asserts its' liveliness, the service
+    deduces that all other entities within the same DomainParticipant are also alive.
+  * **Manual by Topic**: This mode is more restrictive and requires that at least one instance within the DataWriter
+    is asserted to consider that the DataWriter is alive.
+
+.. _liveliness_compatibilityrule:
+
+Compatibility Rule
+""""""""""""""""""
+
+To maintain the compatibility between LivelinessQosPolicy in DataReaders and DataWriters, the DataWriter kind must be
+higher or equal to the DataReader kind.
+And the order between the different kinds is::
+
+ AUTOMATIC < MANUAL BY PARTICIPANT < MANUAL BY TOPIC
```

Comment:
the names should be:
AUTOMATIC_LIVELINESS_QOS
MANUAL_BY_PARTICIPANT_LIVELINESS_QOS
MANUAL_BY_TOPIC_LIVELINESS_QOS